common/src/main/java/com/wynntils/core/features/overlays/TextOverlay.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.overlays;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.overlays.sizes.OverlaySize;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.buffered.BufferedFontRenderer;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import net.minecraft.client.renderer.MultiBufferSource;
+
+/**
+ * An overlay, which main purpose is to display function templates.
+ */
+public abstract class TextOverlay extends Overlay {
+    @Config(key = "overlay.wynntils.textOverlay.textShadow")
+    public TextShadow textShadow = TextShadow.OUTLINE;
```

Comment:
My understanding is that this will toggle shadow for *all* our upcoming text overlay widgets, right? I'm trying to decide if this is a problem or not. (Frankly, I think it has been too many knobs to turn for every individual text setting, so I guess this is not just fine but actually an improvement, but maybe someone will protest..?)

Reply:
per overlay

Reply:
So basically settings are inherited from the base class, but not shared

Reply:
Ok, this sounds good, so I'll accept the PR, but please explain to me then what `key = "overlay.wynntils.textOverlay.textShadow"` means? Aha, that's for the i18n description, not the config save key. Duh. Now I get it. I mixed up the two.